meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.0.0.bb

```diff
@@ -1,12 +1,13 @@
 LICENSE = "CLOSED"
 LICENSE_FLAGS = "commercial"
 
-SUB_FOLDER_arm = "armhf"
+SUB_FOLDER_arm = "arm"
```

Comment:
Curious: Why was it named with the hard-float extension in the first place?

Reply:
We were following Debian naming, but we decided to switch to Yocto.

ref https://github.com/mendersoftware/mender-binary-delta/commit/e9fa48f5cb99bc1a91dcb2d75f6cce77a24f74fc
ref https://www.debian.org/ports/

Reply:
Alright. Thanks